Partially fixing the scripts and documenting them

icyflame opened the pull request:
@VishnuDuttSharma @routeaccess @prudhvid 

I have added documentation to the top of both scripts. (visible in the diff view).
Please see and verify the same.
I have built the package with `rosmake` and the build does pass.

icyflame committed:
Update and rename t.py to stopKraken.py

- Remove blank lines
- Fix comment lines (put a space between `#` and the first word)

icyflame committed:
Update and rename test.py to testThrusters.py

- Remove redundant spaces

icyflame committed:
Remove redundant subscriber code

The node and the subscriber are irrelevant.

icyflame committed:
Fix indentation levels, block comments as per PEP8

- Block comments earlier started with #, but were not followed by a
  space.

icyflame committed:
Document the testThrusters and stopKraken script.

This documentation also contains a TODO section, which must be updated
by anyone who is working on this file.

icyflame commented:
This gets us closer to solving #49 and #50.

prudhvid commented on sensor_stack/seabotix/src/stopKraken.py:
```diff
@@ -0,0 +1,91 @@
+# It creates a handle for the serial port in the Arduino and then writes
+# the data to be fed to the thrusters directly to this serial port.
+
+# It does so using a while loop. (which may be redundant)
```
@VishnuDuttSharma  please confirm this.

prudhvid commented on sensor_stack/seabotix/src/stopKraken.py:
```diff
@@ -0,0 +1,91 @@
+# - Check how the thrusters work
+#     If only publishing once is enough, then the while loop is redundant, 
+#     and we can stop the loop after publishing once.
+# - Write a bash script to start this up directly from the repo root.
```
@icyflame  you can start using rosrun . I don't think you need to write a bash script for this.
Just give execute permissions for this file

icyflame replied:
@prudhvid I just thought that creating a bash script would be much cleaner. And far more easy to understand! I guess that will be fine?

icyflame committed:
Add the init_node call, required for rospy.Rate

icyflame committed:
Ctrl+C handling completed!

- Uses the signal API of python
- On recieving the SIGINT signal, the process runs
stopKraken.py, immediately cutting out the 
thrusters. Pressing Ctrl+C again will stop this script, too, returning
the user to the prompt.

icyflame commented:
@routeaccess @prudhvid 
Completed `testThrusters.py`. Please review and merge.

routeaccess commented:
@icyflame The `SIGINT` was a nice touch. I've been meaning to add it in when this file is refactored, you almost read my mind.
I'm taking your word for this PR - I trust this has been tested on the bot.

icyflame commented:
@routeaccess 
Thank you for the merge!

I have been looking into the `signal` library that Python has, and I am amazed by the functionality it brings to any basic application!

We should start using it in some of the other files, where it may be needed too.
